OrthoEvol/Orthologs/Blast/comparative_genetics.py

```diff
@@ -73,34 +78,62 @@ def __init__(self, project=None, project_path=os.getcwd(), acc_file=None,
         self.__pre_blast = pre_blast
         self.__post_blast = post_blast
         self.__taxon_filename = taxon_file
+        self.acc_file = acc_file
```

Comment:
What's the point of adding another class variable (`self.acc_file`) that is given the same parameter value (`acc_file`) as `self.acc_csv_filename`?

Reply:
In terms of this, `self.acc_file` was being set but at a further downstream point so I moved it here and removed it from that location. I think it's being used in other places in the ComparativeGenetics class. It's mostly just a carryover from the last update to these classes.

I'll take a look at other instances of it and see if I can move this all to one variable.